content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: blog
+title: "CRI-O is moving towards pkgs.k8s.io"
+date: 2023-10-10
+slug: cri-o-community-package-infrastructure
+---
+
+**Author:** Sascha Grunert
+
+The Kubernetes community [recently announced](/blog/2023/08/31/legacy-package-repository-deprecation/)
+that their legacy package repositories are frozen, and now they will move to
+[introduced community-owned packages](/blog/2023/08/15/pkgs-k8s-io-introduction) powered by the
+[openSUSE Build Service (OBS)](https://build.opensuse.org/project/subprojects/isv:kubernetes).
+CRI-O has a long history of utilizing
+[OBS for their package builds](https://github.com/cri-o/cri-o/blob/e292f17/install.md#install-packaged-versions-of-cri-o),
+but all of the packaging efforts have been done manually so far.
+
+The CRI-O community absolutely loves Kubernetes, which means that they're
+delighted to announce that:
+
+**All future CRI-O packages will be shipped as part of the officially supported
+Kubernetes infrastructure hosted on pkgs.k8s.io!**
+
+There will be a deprecation phase for the existing packages, which is currently
+being [discussed in the CRI-O community](https://github.com/cri-o/cri-o/discussions/7315).
+The new infrastructure will only support releases of CRI-O `>= v1.28.2` as well as
+the corresponding release branches.
```

Comment:
> as well as the corresponding release branches.

This is slightly confusing to me.

Reply:
Rephrased to mention the actual release branch.